{{cookiecutter.project_slug}}/.env.example

```diff
@@ -114,6 +114,49 @@ USE_AWS_STORAGE='False'
 #     AWS_LOCATION               Folder location inside of the S3 storage bucket.
 #	  AWS_S3_REGION_NAME		 Region name (ex: 'us-east-2') required in order to receive correct
 #								 query authorization mechanism parameters for private URLs.
+#      ** Due to some recent changes in aws these settings must also be made:
+#       1. create the s3 bucket
+#        2. Uncheck disable Block all public access
+#        3. Add this ACL under Bucket Policy (make sure to specify the path of the folder to allow django access so that private is still private) - Permissions
+#            ```
+#
+#            {
+#                "Version": "2012-10-17",
+#                "Statement": [
+#                    {
+#                        "Sid": "PublicReadGetObject",
+#                        "Effect": "Allow",
+#                        "Principal": "*",
+#                        "Action": "s3:GetObject",
+#                        "Resource": "arn:aws:s3:::<bucket-name>/<folder>/static/*"
+#                    }
+#                ]
+#            }
+#            ```
+#            and in CORS - Permissions
+#            ```
+#                [
+#                    {
+#                        "AllowedHeaders": [
+#                            "Authorization"
+#                        ],
+#                        "AllowedMethods": [
+#                            "GET"
+#                        ],
+#                        "AllowedOrigins": [
+#                            "*"
+#                        ],
+#                        "ExposeHeaders": [
+#                            "x-amz-server-side-encryption",
+#                            "x-amz-request-id",
+#                            "x-amz-id-2"
+#                        ],
+#                        "MaxAgeSeconds": 3000
+#                    }
+#                ]
+#            ```
```

Comment:
This is nice documentation. I do wonder whether this is the most appropriate place for it, but I think it can live here for now, because it's helpful.